### PR TITLE
Include digdag-storage-s3 in CLI

### PR DESCRIPTION
**Pull request by frsyuki**
digdag-storage-s3 was not included in shadow jar because it was
increasing CLI's binary size. However, because digdag-cli already
depends on aws-sdk through digdag-standards, there're no negative
impact any more. Including digdag-storage-s3 increases binary size only
20KB.

### CODE REVIEW

**Comment by frsyuki** [comment]
@komamitsu can I ask you to take a look?

**Comment by komamitsu** [comment]
@frsyuki We don't need to add the dependency to `digdag-cli/build.gradle`?

**Comment by frsyuki** [comment]
@komamitsu It's unnecessary because of this: https://github.com/treasure-data/digdag/pull/450/files#diff-c197962302397baf3a4cc36463dce5eaR338

File: digdag-cli/build.gradle [hunk 1]
```diff
@@ -10,5 +10,8 @@ dependencies {
     compile 'ch.qos.logback:logback-classic:1.1.5'
     compile 'org.fusesource.jansi:jansi:1.11'
     compile 'com.beust:jcommander:1.55'
-    runtime project(':digdag-storage-s3')  // to include digdag-storage-s3 in CLI (shadow jar)
+
+    // This line is for including digdag-storage-s3 in CLI (shadow jar) used through
+    // dynamic class loading. digdag-cli itself doesn't depend on digdag-storage-s3.
+    compile project(':digdag-storage-s3')
```
Comment (komamitsu) [inline digdag-cli/build.gradle]: 👍

**Comment by komamitsu** [comment]
👍

**Pull request merged**